src/libCli/GrammarConstruction.cpp

```diff
@@ -186,21 +186,7 @@ class GrammarInjectorMethodArgs : public GrammarInjector
                     }
                     else
                     {
-                        // FIXME: commented solution does not work:
-                        //        we always complete "::" as empty string matches
-                        //        Solution would be to change the parser to never
-                        //        attempt completion when a regex is currently
-                        //        parsed. This requires changes in the parser
-                        //        and could not be implemented on short notice.
-                        //auto stringContainer = m_grammar.createElement<Concatenation>("StringContainer");
-                        //stringContainer->addChild(m_grammar.createElement<FixedString>(":"));
-                        //stringContainer->addChild(m_grammar.createElement<RegEx>("[^:]*", "FieldValue"));
-                        //stringContainer->addChild(m_grammar.createElement<FixedString>(":"));
-                        //f_fieldGrammar->addChild(stringContainer);
-
-                        // Using this as a workaround until parser gets better regex support
-                        //f_fieldGrammar->addChild(m_grammar.createElement<RegEx>("[^:, ]*", "FieldValue"));
-                        f_fieldGrammar->addChild(m_grammar.createElement<EscapedString>(":, ", ':', "FieldValue"));
+                        f_fieldGrammar->addChild(m_grammar.createElement<EscapedString>(":, %", '%', "FieldValue"));
```

Comment:
A short summary of each of the cases might help to make it easier to understand the addFieldValueGrammar function. Also there is a small typo in l. 118. General question: Why do some comments start with more than two slashes?

Reply:
I added a new commit with improved documentation.

this is "doxygen-style" documentation:

two slashes "//" denote a comment intended for developers working on the code to help understand how the code works and how it is implemented.

three slashes "///" denote documentation of an interface of a function or class which is important for people using the code (without them needing to understand how it is implemented)

In short:
Always use three slashes when you document the function declaration itself, two when you document something inside a function